docs/models.rst

```diff
@@ -93,31 +89,43 @@ default from your declaration.  If you would want to have a one-to-one
 relationship you can pass ``uselist=False`` to
 :func:`~sqlalchemy.orm.relationship`.
 
+Since a person with no name or an email address with no address associated
+makes no sense, ``nullable=False`` tells SQLAlchemy to create the column
+as ``NOT NULL``. This is implied for primary key columns, but it's a good
+idea to specify it for all other columns to make it clear to other people
+working on your code that you did actually want a nullable column and did
+not just forget to add it.
+
 So what do `backref` and `lazy` mean?  `backref` is a simple way to also
 declare a new property on the `Address` class.  You can then also use
 ``my_address.person`` to get to the person at that address.  `lazy` defines
 when SQLAlchemy will load the data from the database:
 
--   ``'select'`` (which is the default) means that SQLAlchemy will load
-    the data as necessary in one go using a standard select statement.
--   ``'joined'`` tells SQLAlchemy to load the relationship in the same
-    query as the parent using a `JOIN` statement.
+-   ``'select'`` / ``True`` (which is the default, but explicit is better
+    than implicit) means that SQLAlchemy will load the data as necessary
+    in one go using a standard select statement.
+-   ``'joined'`` / ``False`` tells SQLAlchemy to load the relationship in
+    the same query as the parent using a `JOIN` statement.
 -   ``'subquery'`` works like ``'joined'`` but instead SQLAlchemy will
     use a subquery.
--   ``'dynamic'`` is special and useful if you have many items.  Instead of
-    loading the items SQLAlchemy will return another query object which
-    you can further refine before loading the items.  This is usually
-    what you want if you expect more than a handful of items for this
-    relationship.
+-   ``'dynamic'`` is special and can be useful if you have many items.
```

Comment:
This might be expanded on to say "if you have many items and typically need a subset of them" or something similar.